src/components/steps/Step2.js

```diff
@@ -101,13 +102,6 @@ function Step2 () {
   }, [wizard])
 
   const checkInput = (type, value) => {
-    if (type === 'teamName' && value.length > 22) {
```

Comment:
Will this validation be done somewhere else?

Reply:
Probably not, but at a later time it could be implemented in the cookiecutter thingy.